cluster/gce/gci/configure-helper-common.sh

```diff
@@ -3301,21 +3233,53 @@ function wait-until-mizar-ready {
   done
 }
 
-function start-mizar-scaleout {
-    echo "Installing Mizar for scale-out architecture..."
-  if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "rp" ]] || [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "node" ]]; then
-    create-mizar-daemon-manifest
-  fi
-  if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "tp" ]]; then
-    create-mizar-operator-manifest
-  fi
+function setup-arktos-mizar-local-admin-conf {
+  cat > /etc/kubernetes/admin.conf <<EOF
+apiVersion: v1
+clusters:
+- cluster:
+    server: http://127.0.0.1:8080
+  name: ${PROJECT_ID}_${CLUSTER_NAME}
+contexts:
+- context:
+    cluster: ${PROJECT_ID}_${CLUSTER_NAME}
+    user: ${PROJECT_ID}_${CLUSTER_NAME}
+  name: ${PROJECT_ID}_${CLUSTER_NAME}
+current-context: ${PROJECT_ID}_${CLUSTER_NAME}
+EOF
 }
 
-function start-mizar-scaleup {
-  #Wait for apiserver to setup default namespace
-  until kubectl get ns | grep default; do
+function start-mizar-scaleout {
+  echo "Installing Mizar for scale-out architecture..."
+  local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
+  mkdir -p "${KUBE_HOME}/mizar"
+
+  echo "Deploying mizar daemonset"
+  # Place mizar daemon yaml.
+  src_file="${src_dir}/mizar-daemon.yaml"
+  sed -i -e "s@{{network_provider_version}}@${NETWORK_PROVIDER_VERSION}@g" "${src_file}"
+  mv "${src_dir}/mizar-daemon.yaml" "${KUBE_HOME}/mizar/"
+  kubectl apply -f "${KUBE_HOME}/mizar/mizar-daemon.yaml"
```

Comment:
start-mizar-scaleout is used in all scaleout tp cluster. It deploys mizar daemon as well as crds and operators. The later two can be deployed to all tps but not daemon. Is there anything prevent mizar daemon from being deployed more than one tp?

Reply:
Daomon can be deployed only from tp1 if this is the agreement.